### PR TITLE
Feature/camera drag

### PR DESCRIPTION
**Pull request by dabeshouse**

### CODE REVIEW

**Review by dportnoy15 — APPROVED** [review]

File: core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java [hunk 1]
```diff
@@ -304,6 +304,10 @@ public Visual[] getSgVisuals() {
     return isVrActive ? sgVrVisuals : sgCameraVisuals;
   }
```
Comment (jennej) [inline core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java:306]: i have no idea what putInstance does.  might warrant a comment?

File: core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java [hunk 2]
```diff
@@ -595,8 +596,9 @@ private void handleManipulatorSelection(SelectionEvent e) {
         } else {
           this.setSelectedField(field.getDeclaringType(), field);
         }
```
Comment (jennej) [inline core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java:598]: this function is starting to get scary, but it is also creeping into drag adaptor territory, so i think i'm backing away quietly to not wake the beast.

**Review by jennej — APPROVED** [review]

File: core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java (hunk 1 above)
Comment (dabeshouse) [reply core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java:306]: It is a scary subsystem on every Element that I want to remove, but today it is useful.
A comment has been added to the code too.

File: core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java (hunk 2 above)
Comment (dabeshouse) [reply core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java:598]: So many dispatcher methods like this.
More "Uh oh" than "OO", amirite?

I join you in slowly stepping away.

**Pull request merged**